swagger.yaml

```diff
@@ -35,25 +42,108 @@ paths:
           description: "Searches class title, description, and subject"
           required: false
           type: string
-        - in: query
-          name: page[size]
-          description: "Number of results to return. Used in pagination. Default value is 10."
-          required: false
-          type: string
-        - in: query
-          name: page[number]
-          description: "Page number of results. Used to paginate through results."
-          required: false
+        - $ref: '#/parameters/pageSize'
+        - $ref: '#/parameters/pageNumber'
+        - $ref: '#/parameters/pretty'
+      responses:
+        "200":
+          description: "Successful response"
+          schema:
+            $ref: "#/definitions/CoursesResultObject"
+        "404":
```

Comment:
will this ever return a `404`? Same with `GET /subject`, `GET /terms`

Reply:
good point, they won't return 404's